proposals/proposal_template.md

```diff
@@ -0,0 +1,140 @@
+# Proposal: &lt;API>
+
+**Summary**
+
+An API proposal for your new API.  Describe in **one sentence** what it will be
+used for.
+
+**Document Metadata**
+
+**Author:** &lt;your Github handle>
+
+**Sponsoring Browser:** The browser vendor committed to implementing this API
+if approved.
+
+**Contributors:** &lt;other contributors @>
+
+**Created:** YYYY-MM-DD
+
+**Related Issues:** &lt;urls>
+
+## Motivation
+
+### Objective
+
+What does this API enable?  Why do we need it?
+
+#### Use Cases
+
+Describe the use cases for this API.  There may be multiple.
+
+### Known Consumers
+
+APIs should, ideally, have significant adoption once they are complete.
+Highlight any extensions that have committed to using this API.
+
+## Specification
+
+### Schema
+
+Include your API schema here (preferably with a TypeScript interface).  If you
+are adding new functionality or fields to an existing method, please be sure to
+call out explicitly what has changed.
+
+A few notes:
+
+*   Extension APIs are almost always asynchronous.  Anything that cannot be
+    resolved directly in the renderer must be asynchronous, and even if
+    something can be done in the renderer today, it should frequently be
+    asynchronous to prevent future breakage if we move something to the browser
```

Comment:
```suggestion
    resolved directly in a child process must be asynchronous, and even if
    something can be done in the child process today, it should frequently be
    asynchronous to prevent future breakage if we move something to the parent
```

"renderer"/"browser" is Chromium-specific terminology.

I replaced it with the more general "child"/"parent" process terminology. Instead of "parent", "different" could also be used.

Reply:
Rephrased, and also added details that extensions usually run in child processes.